src/modules/advanced-block-spacing/index.js

```diff
@@ -735,6 +735,8 @@ const advancedBlockSpacing = ( blockName, options = {} ) => {
 		enablePaddingRight: true,
 		enablePaddingBottom: true,
 		enablePaddingLeft: true,
+		overwritePaddingUnits: false,
+		overwriteMarginUnits: false,
```

Comment:
Instead of doing a default value of false for `overwritePaddingUnits`, we can use `[ 'px', 'em', '%'  ]` as the default value for `overwritePaddingUnits` and then pass it directly to the Control